sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/DigitalTwinsRelationshipAsyncTest.java

```diff
@@ -0,0 +1,276 @@
+                    }
+                )
+                .verifyComplete();
+
+            // Create relationship from Floor -> Hvac
+            StepVerifier
+                .create(asyncClient.createRelationship(floorTwinId, FLOOR_COOLED_BY_HVAC_RELATIONSHIP_ID, floorCooledByHvacPayload, BasicRelationship.class))
+                .assertNext(
+                    basicRelationship -> {
+                        assertThat(basicRelationship.getId())
+                            .isEqualTo(FLOOR_COOLED_BY_HVAC_RELATIONSHIP_ID)
+                            .as("Created relationship from floor -> hvac");
+                        logger.info("Created {} relationship between source = {} and target = {}", basicRelationship.getId(), basicRelationship.getSourceId(), basicRelationship.getTargetId());
+                    }
+                )
+                .verifyComplete();
+
+            // Create relationship from Hvac -> Floor
+            StepVerifier
+                .create(asyncClient.createRelationship(hvacTwinId, HVAC_COOLS_FLOOR_RELATIONSHIP_ID, floorTwinCoolsRelationshipPayload, BasicRelationship.class))
+                .assertNext(
+                    basicRelationship -> {
+                        assertThat(basicRelationship.getId())
+                            .isEqualTo(HVAC_COOLS_FLOOR_RELATIONSHIP_ID)
+                            .as("Created relationship from hvac -> floor");
+                        logger.info("Created {} relationship between source = {} and target = {}", basicRelationship.getId(), basicRelationship.getSourceId(), basicRelationship.getTargetId());
+                    }
+                )
+                .verifyComplete();
+
+            // Create relationship from Room -> Floor
+            StepVerifier
+                .create(asyncClient.createRelationship(roomTwinId, ROOM_CONTAINED_IN_FLOOR_RELATIONSHIP_ID, floorTwinContainedInRelationshipPayload, BasicRelationship.class))
+                .assertNext(
+                    basicRelationship -> {
+                        assertThat(basicRelationship.getId())
+                            .isEqualTo(ROOM_CONTAINED_IN_FLOOR_RELATIONSHIP_ID)
+                            .as("Created relationship from room -> floor");
+                        logger.info("Created {} relationship between source = {} and target = {}", basicRelationship.getId(), basicRelationship.getSourceId(), basicRelationship.getTargetId());
+                    }
+                )
+                .verifyComplete();
+
+            // Create a relation which already exists - should return status code 409 (Conflict).
+            StepVerifier.create(asyncClient.createRelationship(roomTwinId, ROOM_CONTAINED_IN_FLOOR_RELATIONSHIP_ID, floorTwinContainedInRelationshipPayload))
+                .verifyErrorSatisfies(ex -> assertRestException(ex, HTTP_PRECON_FAILED));
+
+            // Update relationships
+
```

Comment:
Like in the sync client test, are you missing code here?

Reply:
No, the update operation is in the line right below this comment. Is the additional spacing confusing? Or is it because of the comment "**Create** relationship from Floor -> Room"?